### PR TITLE
feat: add no-bind rule

### PR DESCRIPTION
**Pull request by nemmip**
#### Pull Request purpose

[ ] Documentation update
[ ] Bug fix
[X] New rule
[ ] Updates an existing rule
[ ] Add something to the core
[ ] Other, please explain:

#### Checklist

[ ] Update CHANGELOG.md
[X] Update task status in release plan (README)
[ ] Docs added (/docs/rules/<rule_name>)
[ ] Docs ready to be added to website (ec0lint.com)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,7 @@ Let's build an eco-friendly website!
 - :ballot_box_with_check: Light libraries - jQuery andSelf (no-and-self, ec0lint)
 - :hammer: Light libraries – jQuery Animate (no-animate, ec0lint)
 - :hammer: Light libraries – jQuery Attr (no-attr, ec0lint)
+- :hammer: Light libraries – jQuery Bind (no-bind, ec0lint)
```
Comment (malwinq) [inline README.md]: Pls change done jQuery rules to blue checkbox

File: docs/rules/no-bind.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# no-bind
+
+Disallows the [`.bind`](https://api.jquery.com/bind/)/[`.unbind`](https://api.jquery.com/unbind/) methods. Prefer `.on`/`.off` or `EventTarget#addEventListener`/`removeEventListener`.
+
+📋 This rule is enabled in `plugin:no-jquery/deprecated-3.0`.
```
Comment (malwinq) [inline docs/rules/no-bind.md]: I think we're a different plugin now, so the release log can be deleted

File: lib/rules/index.js [hunk 3]
```diff
@@ -15,8 +15,7 @@ module.exports = new LazyLoadingRuleMap(Object.entries({
     "no-ajax": () => require("./no-ajax"),
     "no-ajax-events": () => require("./no-ajax-events"),
     "no-and-self": () => require("./no-and-self"),
-    "no-ajax": () => require("./no-ajax"),
-    "no-ajax-events": () => require("./no-ajax-events"),
```
Comment (malwinq) [inline lib/rules/index.js]: Two rules are deleted

File: lib/rules/no-bind.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+                    return;
+                }
+                const name = node.callee.property.name;
+                if (!methods.includes(name)) {
+                    return;
+                }
+                if (utils.isjQuery(context, node.callee)) {
+                    context.report({
+                        node: node,
+                        message: 'Prefer `.on`/`.off` or `EventTarget#addEventListener`/`removeEventListener`',
```
Comment (malwinq) [inline lib/rules/no-bind.js]: Also the log message can be changed - lets discuss

File: lib/rules/index.js (hunk 3 above)
Comment (nemmip) [reply lib/rules/index.js]: bc they are duplicates (see line 15-16)

File: lib/rules/no-bind.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+/** @type {import('../shared/types').Rule} */
+module.exports ={
+    meta: {
+        type: 'suggestion',
+        docs: {
+            description: 'Disallows bind utilities: bind, unbind. Prefer `.on`/`.off` or `EventTarget#addEventListener`/`removeEventListener`',
+            category: "Code improvements",
+            recommended: false,
+            url: "https://eslint.org/docs/rules/no-bind"
```
Comment (malwinq) [inline lib/rules/no-bind.js]: change url to ec0lint (works after Ola's change)

**Review by malwinq — APPROVED** [review]

**Pull request merged**